app/components/video-player.jsx

```diff
@@ -84,14 +91,19 @@ const VideoPlayer = React.createClass({
         <span>
           {rate}&times;
         </span>
-      </label>)
+      </label>),
     );
   },
 
   render() {
     const rates = [0.25, 0.5, 1];
     return (
       <div className="subject-video-frame">
+        <button
+          className="subject-video-frame__hidden-button"
+          onClick={this.playVideo.bind(this, !this.state.playing)}
+        />
+
         <video
           className="subject"
           ref="videoPlayer"
```

Comment:
The idea was to hide all controls for the video element, including native controls. This was to adhere to styling for the new project landing page.